lib/logstash/outputs/elasticsearch/http_client.rb

```diff
@@ -135,12 +136,16 @@ def join_bulk_responses(bulk_responses)
     def bulk_send(body_stream)
       params = http_compression ? {:headers => {"Content-Encoding" => "gzip"}} : {}
       # Discard the URL 
+      @metric.increment(:bulk_requests)
+      @metric.increment(:concurrent_bulk_requests)
```

Comment:
From what I tested locally, the value of this metric was either 0 or `pipeline.workers`. Because my pipelines were always light on processing, the bottle neck was ES, hence the value == `pipeline.workers`. However alt+tabbing and performing other tasks in my laptop would make this value become 0 temporarily.

I am under the impression that in production it's rare for this value to be anything other than 0-1. Otherwise it would mean the pipeline is under constant backpressure if the event rate is > Batch Size / Time per back, or  in this case 125/ 0.051s = 2450 eps